### ISSUE TITLE
CTRL/ESC Keys break in kitty

### ISSUE THREAD

**Issue by trev-dev** [neomutt/neomutt#3393]
I am using kitty terminal. This problem does not happen in other terminals, but it is unique to neomutt in the sense that this is where I get hung up, so I felt perhaps this is something another neomutter might already know how to fix.

In Kitty, if I move from neomutt to neovim and back, my ESC and CTRL key commands stop sending and get entered as text input.

This makes it so that I cannot break out of prompts or certain menus:
![brokemutt](https://user-images.githubusercontent.com/28788713/166262194-2250e492-a5be-4891-a4c2-82e4f485d610.png)

To get out I have to kill the terminal emulator and go back in. So silly! Any ideas?

**Comment by kovidgoyal** [comment]
That will be a bug in neovim turning on the kitty extended keybaor dprotocol and not turning it off again on exit.

**Comment by trev-dev** [comment]
> That will be a bug in neovim turning on the kitty extended keybaor dprotocol and not turning it off again on exit.

 Thanks for this. I'm struggling to find a confuration/man page about this feature so I can write my way out of this problem or confidently narrow it down.

 I really like Kitty. Is there more I can do to demonstrate/make this bug re-producable?

**Comment by kovidgoyal** [comment]
It's already fixed in neovim master. If you are interested search its
issue tracker for kitty.

**Comment by flatcap** [comment]
@kovidgoyal
Thanks for stepping in!
It had me really worried if this was a NeoMutt bug :-)

**Comment by trev-dev** [comment]
> It's already fixed in neovim master. If you are interested search its issue tracker for kitty.

> @kovidgoyal
> Thanks for stepping in!
> It had me really worried if this was a NeoMutt bug :-)

Yes, thank you! I am on nightly. I imagine I will see this fix soon enough :)
